Header header

int32 Counter
int8 StateCode
int32 StateData
float32 Distance
float32 Movement
int32 SignalQuality
